src/Forms/HTMLEditor/HTMLEditorSanitiser.php

```diff
@@ -331,6 +346,32 @@ public function sanitise(HTMLValue $html)
                     $el->setAttribute($attr, $forced);
                 }
             }
+
+            if ($el->tagName === 'a' && $linkRelValue !== null) {
+                $this->addRelValue($el, $linkRelValue);
+            }
+        }
+    }
+
+    /**
+     * Adds rel="noopener noreferrer" to link elements with a target attribute
+     *
+     * @param DOMElement $el
+     * @param string|null $linkRelValue
+     */
+    private function addRelValue(DOMElement $el, $linkRelValue)
+    {
+        // user has checked the checkbox 'open link in new window'
+        if ($el->getAttribute('target') && $el->getAttribute('rel') !== $linkRelValue) {
+            if ($linkRelValue !== '') {
+                $el->setAttribute('rel', $linkRelValue);
+            } else {
+                $el->removeAttribute('rel');
+            }
+        } elseif ($el->getAttribute('rel') === $linkRelValue && !$el->getAttribute('target')) {
```

Comment:
So the way you could avoid an elseif here is simply to `return` after the previous else clause, then this can just be a normal `if`